src/components/OperationRow.js

```diff
@@ -46,24 +47,28 @@ class OperationRow extends PureComponent<Props, *> {
     displayCurrencyLogo: false,
   };
 
-  goToOperationDetails = () => {
-    const {
-      navigation,
-      account,
-      parentAccount,
-      operation,
-      isSubOperation,
-    } = this.props;
-    const params = {
-      accountId: account.id,
-      parentId: parentAccount && parentAccount.id,
-      operation, // FIXME we should pass a operationId instead because data can changes over time.
-      isSubOperation,
-      key: operation.id,
-    };
+  goToOperationDetails = debounce(
+    () => {
+      const {
+        navigation,
+        account,
+        parentAccount,
+        operation,
+        isSubOperation,
+      } = this.props;
+      const params = {
+        accountId: account.id,
+        parentId: parentAccount && parentAccount.id,
+        operation, // FIXME we should pass a operationId instead because data can changes over time.
+        isSubOperation,
+        key: operation.id,
```

Comment:
Maybe retrieving the current operation Id from the navigation object and checking that it will be different from the one we want to push should suffice in this case. 🤷‍♂️
Sounds weird to me that react-navigation allows that kind of behaviour there might be a global fix for this also. 😅